File: 프로그래머스/1/131705. 삼총사/삼총사.js
function solution(n) {
    var answer = 0;
    let len = n.length;
    for(let i = 0; i < len-2; i++){
        for(let j = i+1; j < len-1; j++){
            for(let k = j+1; k < len; k++){
                if(n[i]+n[j]+n[k] === 0){
                    answer++;
                }
            }
        }
    }
    return answer;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    ans = 0
    for i in range(len(number)-2):
        for j in range(i+1, len(number)-1):
            for l in range(j+1, len(number)):
                if number[i] + number[j] + number[l] == 0:
                    ans += 1
                    
    return ans